soh/soh/config/ConfigMigrators.h

```diff
@@ -299,12 +299,8 @@ namespace SOH {
         { MigrationAction::Rename, "gUniformLR", "gEnhancements.FixMenuLR" },
         { MigrationAction::Rename, "gVisualAgony", "gEnhancements.VisualAgony" },
         { MigrationAction::Rename, "gVoidDamageMul", "gEnhancements.VoidDamageMult" },
-        { MigrationAction::Rename, "gGameplayStats.ShowAdditionalTimers", "gEnhancements.GameplayStats.ShowAdditionalTimers" },
-        { MigrationAction::Rename, "gGameplayStats.ShowDebugInfo", "gEnhancements.GameplayStats.ShowDebugInfo" },
-        { MigrationAction::Rename, "gGameplayStats.RoomBreakdown", "gEnhancements.GameplayStats.RoomBreakdown" },
-        { MigrationAction::Rename, "gGameplayStats.ShowIngameTimer", "gEnhancements.GameplayStats.ShowInGameTimer" },
-        { MigrationAction::Rename, "gGameplayStats.TimestampsReverse", "gEnhancements.GameplayStats.ReverseTimestamps" },
-        { MigrationAction::Rename, "gGameplayStats.RTATiming", "gEnhancements.GameplayStats.RTATiming" },
```

Comment:
Shouldn't all of these get the migration treatment?

Reply:
A couple of them do, but after the change, ShowAdditionalTimers, ShowDebugInfo, RoomBreakdown, and RTATiming actually ended up having the exact same name before and after migration, so there would be no point in migrating them.

Reply:
Oh, I didn't even think of that. Yup fair enough.